### PR TITLE
make script to upload historical stats from csv

### PR DESCRIPTION
**Pull request by lukejolliffe**
# Purpose

Write rough script to write initial stats to table from spreadsheet. 

Fixes UML-2682

## Checklist

* [x] I have performed a self-review of my own code
* [x] I have added relevant logging with appropriate levels to my code

### CODE REVIEW

File: scripts/add_historical_statistics/stats.csv [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+forbidden_login_attempt_failures,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+not_found_login_attempt_failures,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+activation_key_exists_event,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+activation_key_expired_event,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+activation_key_not_exists_event,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+actor_code_failure_rate_limit_event,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+actor_login_failure_rate_limit_event,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+identity_hash_change_event,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+viewer_code_failure_rate_limit_event,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (jamesrwarren) [inline scripts/add_historical_statistics/stats.csv:51]: whats with these lines?

Comment (lukejolliffe) [reply scripts/add_historical_statistics/stats.csv]: Just spreadsheet -> csv things. Gone now :)

**Review by jamesrwarren — APPROVED** [review]
Nice :)

**Review by cooperaj — COMMENTED** [review]
👍

**Pull request merged**